src/index.js

```diff
@@ -389,7 +403,29 @@ const childrenColumnCheck = (props, _, componentName) => {
 }
 
 Table.propTypes = {
-  classes: PropTypes.object,
+  classes: PropTypes.shape({
```

Comment:
You can just use this if all the items in are of the same type

`classes: PropTypes.objectOf(PropTypes.string)`

Reply:
will do next time thanks!